Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -39,22 +39,27 @@ class DataPipelineImplementation: DataPipelineInstance {
     var config: CioInternalCommon.SdkConfig?
 
     func identify(identifier: String, body: [String: Any]) {
-        analytics.identify(userId: identifier, traits: body)
+        commonIdentifyProfile(userId: identifier, attributesDict: body)
     }
 
     func identify<RequestBody: Codable>(identifier: String, body: RequestBody) {
-        analytics.identify(userId: identifier, traits: body)
+        commonIdentifyProfile(userId: identifier, attributesCodable: body)
+    }
```

Comment:
Some `identify` calls are being sent directly by CustomerIO+Segment extensions, would we add these in here as well?

Reply:
Discussed on call, will try using plugin or routing all through DataPipeline, whatever works